src/Enums/PolygonMaskType.ts

```diff
@@ -0,0 +1,8 @@
+"use strict";
+
+export enum PolygonMaskType {
+    none = "none",
+    inside = "inside",
+    outside = "outside",
+    inline = "inline",
+}
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [file should end with a newline](https://app.codacy.com/manual/ar3s/tsparticles/pullRequest?prid=5042590)